### PR TITLE
chore(test): drop usage of ganache in old E2E tests

### PR DESCRIPTION
**Pull request by jakubgs**
The Ganache tool has been replaced by Hardhat:
https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat

Also, yes, I don't use PR templates, call the police.

### CODE REVIEW

**Comment by anastasiyaig** [comment]
image

**Review by anastasiyaig — APPROVED** [review]

File: ci/Jenkinsfile.tests-e2e [hunk 1]
```diff
@@ -97,44 +94,27 @@ pipeline {
...
     }
 
-    stage('Containers') {
-      parallel {
-        stage('Ganache') { steps { script {
-          ganache = docker.image(
-            'trufflesuite/ganache:v7.4.1'
-          ).run(
-            ["-p 127.0.0.1:${env.GANACHE_RPC_PORT}:8545",
-             "-v ${env.WORKSPACE}/test/ui-test/fixtures/ganache-dbs/goerli:/goerli-db"].join(' '),
```
Comment (stefandunca) [inline ci/Jenkinsfile.tests-e2e:115]: @anastasiyaig do the testing team plans to migrate the text fixture to the new tool or we should remove the ganache DB and data folder also?

**Review by stefandunca — APPROVED** [review]
👌🏼

File: ci/Jenkinsfile.tests-e2e (hunk 1 above)
Comment (anastasiyaig) [reply ci/Jenkinsfile.tests-e2e:115]: we want to try other solution, but ganache is dead completely so lets remove everything

**Pull request merged**